politeiad/backend/gitbe/decred.go

```diff
@@ -34,8 +34,9 @@ import (
 // XXX plugins really need to become an interface. Run with this for now.
 
 const (
-	decredPluginIdentity = "fullidentity"
-	decredPluginJournals = "journals"
+	decredPluginIdentity     = "fullidentity"
+	decredPluginJournals     = "journals"
+	decredPluginInventoryCmd = "inventorycmd"
```

Comment:
Why is this postfixed cmd?

Reply:
It doesn't need to be.  It has been shortened to "inventory".